### DISCUSSION TITLE
Using TogetherSweep() with context measurement?

### DISCUSSION THREAD

**Discussion by mapmen** [microsoft/Qcodes#6354] [Q&A] [answered]
Is is possible to use TogetherSweep() outside dond? Like, using it with context measurement? Something like:

with context_meas.run() as datasaver:    
        sweep = TogetherSweep(param_1, param_2) ......

**Comment by jenshnielsen** [comment, marked as answer]
That is not currently something that is implemented but you could probably write a function that took a TogetherSweep and did something with it
